src/utils/isReactComponent.js

```diff
@@ -1,4 +1,5 @@
 export default function isReactComponent (component) {
+  if (!component) return false
```

Comment:
Perhaps it would be better to write this as 
```js
if (component === undefined || component === null || component === false)
```
For example, I'm not sure if this is correct behaviour for cases when `component` is `''` or `0`.

Reply:
Hey, thanks for your reply. I think `''` and `0` shouldn't be considered valid React components either so it might be ok to leave it as is?

Reply:
The thing is, it works both ways because the React wrapper is also using this function. Actually, with your fix it's the React side that will suffer from similar errors. Instead, we should probably just leave "falsy" components as they were. Shouldn't be difficult to do, do you think you can find the time to implement this?

Reply:
To be clear, I meant not wrapping falsy components whatsoever.